Training 441 Crawl/css/styles.css

```diff
@@ -0,0 +1,84 @@
+/*!
+ * All Rights Reserved
+ * This software is proprietary information of
+ * Intelligent Sense
+ * Use is subject to license terms.
+ * Filename: style.css
+ */
+/*
+ * Author:      fquiros@akurey.com
+ * Date:        30/07/2019
+ * Description: Styles for training 441.
+ */
+
+
+@import url('https://fonts.googleapis.com/css?family=Acme'); 
+
+
+
+.main-container{
+    text-align: center;  
+    margin: 0 auto;
+}
+
+.title{
+    text-align: center;  
+    
+}
+.box{
+    color: white;
+    font-size: 16px;
+    text-align: left;
+    display: inline-block;
+    width: 15em;
+    height: 20em;
+    margin: 2rem;
+    padding: 1%;
+ }
+
+.facebook{
+    background: rgb(59, 89, 152);
+
+}
+
+.facebook--font{
+    font-family: 'Franklin Gothic Medium';
+}
+
+.facebook__icon{
+    text-align: center;
+    margin-top: 65%;
+}
+
+.twitter{
+    background: rgb(92, 185, 211);
+
+}
+
+.twitter--font{
```

Comment:
--font suffix doesn't look like a good BEM name convention practice. Instead rename it to be `.twitter__paragraph {}`. The same applies to the others (facebook, instagram...). Also not needed to put a style to specific element, you can just put it in `.twitter{}`. In less selectors you use, more efficient your code will be.